.github/workflows/pr-lints.yml

```
@@ -0,0 +1,24 @@
+name: "github"
+
+on:
+  pull_request:
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.7.x"
```

Modernize today; it'll stay frozen for years:

```suggestion
          python-version: "3.9.x"
```